README.md

```diff
@@ -36,4 +52,22 @@ If you want to disable pylint, you can [disable this extension](https://code.vis
 
 ## Logging
 
-From the command palette (View > Command Palette ...), run the `Developer: Set Log Level...` command. From the quick pick menu, select `Pylint` extension from the `Extension logs` group. Then select the log level you want to set.
+From the Command Palette (**View** > **Command Palette ...**), run the **Developer: Set Log Level...** command. Select **Pylint** from the **Extension logs** group. Then select the log level you want to set.
+
+Alternatively, you can set the `pylint.trace.server` setting to `verbose` to get more detailed logs from the Pylint server. This can be helpful when filing bug reports.
+
+To open the logs, click on the language status icon (`{}`) on the bottom right of the Status bar, next to the Python language mode. Locate the **Pylint** entry and select **Open logs**.
+
+
+## Troubleshooting
+
+In this section, you will find some common issues you might encounter and how to resolve them. If you are experiencing any issues that are not covered here, please [file an issue](https://github.com/microsoft/vscode-pylint/issues).
+
+- If the `pylint.importStrategy` setting is set to `fromEnvironment` but Pylint is not found in the selected environment, this extension will fallback to using the Pylint binary that is shipped with the extension. However, if there are dependencies installed in the environment, those dependencies will be used along with the shipped Pylint binary. This can lead to problems if the dependencies are not compatible with the shipped Pylint binary.
+
+    To resolve this issue, you can:
+
+    - Set the `pylint.importStrategy` setting to `useBundled` and the `pylint.path` setting to point to the custom binary of Pylint you want to use; or
+    - Install Pylint in the selected environment.
+
+- If you are using a custom Pylint binary, make sure that the binary is compatible with the version of Python you have selected in your workspace. You can check the version of Python used by the extension by running the **Python: Select Interpreter** command from the Command Palette (**View** > **Command Palette...**).
```

Comment:
Actually this is not a restriction. if you are using pylint binary, as long as we can parse the output we can use it. This is a way to allow backwards compatibility.